spec/dummy/app/models/user.rb

```diff
@@ -97,8 +99,4 @@ def permissions
       delete: true,
     }
   end
-
-  def permissions=(value)
-    # assert here the value is a hash and has the right values sent in by the user
-  end
```

Comment:
@adrianthedev I thought it'd be nicer to not have a body for this at all and just have this method created via the setter. WDYT?

Reply:
Oh, so basically set the value on the model instance class. 
Yeah, that could work too. 
Please as a comment on the writer why it's there.